documentation/development-docs/examples/private-link-resource-example.md

```diff
@@ -71,6 +75,8 @@ New-AzPrivateEndpoint -ResourceGroupName {rg_name} -Name {endpoint_name} -Locati
 ```
 
 ### step-by-step
+0. Add `Network.csproj` to {Module}.sln, and `Microsoft.Azure.Management.Network` to {Module}.Test.csproj
```

Comment:
Use bullet mark down instead of number. otherwise, you have to maintain the order everytime.